docs/components/bottom_sheets.md

```diff
@@ -0,0 +1,286 @@
+
+```dart
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Standard Bottom Sheet Demo',
+      home: Scaffold(
+        // Use a Builder to get a context within the Scaffold.
+        body: Builder(
+          builder: (context) {
+            return Center(
+              child: TextButton(
+                child: Text('SHOW BOTTOM SHEET'),
+                onPressed: () {
+                  showModalBottomSheet(
+                    context: context,
+                    builder: (context) {
+                      // Using Wrap makes the bottom sheet height the height of the content.
+                      // Otherwise, the height will be half the height of the screen.
+                      return Wrap(
+                        children: [
+                          ListTile(
+                            leading: Icon(Icons.share),
+                            title: Text('Share'),
+                          ),
+                          ListTile(
+                            leading: Icon(Icons.link),
+                            title: Text('Get link'),
+                          ),
+                          ListTile(
+                            leading: Icon(Icons.edit),
+                            title: Text('Edit name'),
+                          ),
+                          ListTile(
+                            leading: Icon(Icons.delete),
+                            title: Text('Delete collection'),
+                          ),
+                        ],
+                      );
+                    },
+                  );
+                },
+              ),
+            );
+          },
+        ),
+      ),
+    );
+  }
+}
+```
+
+## Anatomy and key properties
+
+The following shows the anatomy of a modal bottom sheet:
+
+![Modal bottom sheet anatomy](assets/bottom_sheets/modal-anatomy.png)
+
+1. Sheet
+1. Contents
+1. Scrim
+
+
+#### Sheets properties
+
+&nbsp;         | Property
+-------------- | ------------------------
+| **Color** | `backgroundColor` |
+| **Elevation** | `elevation` |
+| **Shape** | `shape` |
+
+#### Contents properties
+
+There are no specific properties for content because the content can be any composition of widgets.
+
+#### Scrim properties
+
+&nbsp;         | Property
+-------------- | ------------------------
+| **Color** | `barrierColor` |
+
+
+## Expanding bottom sheet
+
+Expanding bottom sheets require creating a custom widget. See [Expanding bottom sheet](link) for more info.
+TODO: link https://github.com/material-components/material-components-flutter/blob/develop/docs/components/expanding-bottom-sheet.md on material.io
+
+## Theming bottom sheets
+
+Bottom Sheets support [Material Theming](https://material.io/components/sheets-bottom#theming) and can be customized in terms of color, elevation and shape.
```

Comment:
Why isn't there example code/screenshot?

Reply:
The shrine study doesn't show what to do for the shape or elevation or background color. Applying the shrine theme as we do in other components makes no change, so the only change here would be to modify the BottomSheetThemeData with some shape, elevation, or color.

Reply:
Though Shrine text is brown, and the standard sheet would have a pink header with brown text. Can we switch to using the standard example except in Shrine?

Reply:
Ah good point, updated with standard example in shrine theme